src/App.js

```diff
@@ -1,82 +0,0 @@
-
-import robot from './Components/robot.png';
-import './App.css';
-import Profile from './Components/Profile'
-
-import react,{useState,useEffect} from 'react';
-import axios from 'axios'
-
-function App() {
-  const [Data,setData]=useState([]);
-  const [search,setSearch]=useState('');
-  const [searchResult,setSearchResults] = useState("");
-
- useEffect(()=>{
-   axios.get('https://jsonplaceholder.typicode.com/users')
-   .then(response => {
-     setData(response.data);
-     
-    });
- },[]) ;
-
- const dataFilter =(e)=>{
-  setSearch(e.target.value)
```

Comment:
I dont think you need search and searchResult

Reply:
search is user input value, searchResult is the filtered data which is shown on the screen.